<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartDisplayer</class>
 <widget class="QWidget" name="ChartDisplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>答题情况</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <widget class="QLabel" name="pic_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="barChart_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>题型正确率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pieChart_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>题量分布</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lineChart_btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正确率趋势</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
